_posts/2023-12-01-november-2023.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: November 2023
+author: Bestem0r
+---
+
+November er over og den uungåelege vinterkulda og mørkret har omsider lagt seg over Trondheim by 🥶. I dei tusen studenthyblar sit byens studentar djupt konsentrerte over bøker og notat i førebuinga til eksamensperioden me nå går inn i.  
+Er du kanskje ein av dei som treng eit lite avbrekk frå eksamenslesinga og det depressive vintervêret?
+I så fall kan eg glede deg med at nyaste utgåve av utviklarbloggen er her! Lat oss sjå på kva webkom har jobba med i november! 🎉
+
+<br>
+## Individuelle emojis på møteinnkallingar
+Har du nokon gong lurt på kven som har reagert med kva emoji på møteinnkallingar? 🤔
+No kan du endeleg sjå det!
+Ved å halde musepeikaren over emoji-reaksjonen vil du få opp ei liste over kven som har reagert med den gitte emoji-en. 🥳
+
+| Før                                                | Etter |
+|----------------------------------------------------| --- |
+| ![Før](/images/posts/2023-12-02-emojis-before.png) | ![Etter](/images/posts/2023-12-02-emojis-after.png) |
+
+## Referent på møte
+
+Når du opprettar eit møte kan du no legge til ein referent. 📝
```

Comment:
Maybe specity that this is while creating a meeting and inviting a group. We have had "referent" for a while, just a bit buggy while creating the meeting:)